.changeset/tall-gorillas-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/nextjs': minor
```

Comment:
this package is still on 0. major version so this non-breaking addition can be done with a patch.

```suggestion
'@lit-labs/nextjs': patch
```